tests/Graphics/Image/ProcessingSpec.hs

```diff
@@ -13,17 +13,22 @@ import Graphics.Image.InterfaceSpec (translateWrap, dummyImage10x20)
 data Interpol
   = I1 Nearest
   | I2 Bilinear
+  | I3 Bicubic
 
 instance Show Interpol where
   show (I1 i) = "I1 " ++ show i
   show (I2 i) = "I2 " ++ show i
+  show (I3 i) = "I3 " ++ show i
 
 instance Arbitrary Interpol where
   arbitrary = do
     ix <- arbitrary
-    case ix `mod` (2 :: Int) of
+    case ix `mod` (3 :: Int) of
       0 -> return $ I1 Nearest
       1 -> return $ I2 Bilinear
+      2 -> do
+        Negative a <- arbitrary
```

Comment:
I think it would make sense to generate `a` in range `[-1, -0.5]`, since that is expected range, something like:
```haskell
  a <- choose (-1, -0.5)
  ...
```
but if you choose to leave it like it is now it's fine by me.

Reply:
I will rewrite it too.